packages/terra-button/src/_mixins.scss

```diff
@@ -75,3 +75,65 @@
     color: map-get($color-map, disabled-color);
   }
 }
+
+@mixin terra-consumer-theme-button($color) {
+  $color-dark-30: mix($color, #000, 70%);
```

Comment:
since they are local sass variables, it's a bit tricky to put them in a mixin. Instead I would keep them in `terra-consumer-theme-button` and extends the params of this mixin to match the default theme. And then remove the mixin `terra-consumer-theme-button-default`. So it would be:
```
@mixin terra-consumer-theme-button($color, $color1, $color2) {
  $color-dark-30: mix($color, $color1, 70%);
  $color-dark-50: mix($color, $color1, 50%);
  $color-light-30: rgba($color2, 0.3);
  $color-light-50: rgba($color, 0.5);
  $color-light-65: rgba($color2, 0.65);

  --terra-button-background-color-primary: $color;
  --terra-button-border-color-primary: $color;
  --terra-button-hover-background-color-primary: $color-dark-30;
  ...
}

@include cerner-consumer-theme {
  @include terra-consumer-theme-button(#64696c, #1c1f21, #383f42);

  /* stylelint-disable selector-class-pattern */
  .terra-consumer-theme-concept1 {
    @include terra-consumer-theme-button(#c24746, #000, #c24746);
  }
}
```